scripts/circle-ci/run-circle-tests.sh

```diff
@@ -2,7 +2,7 @@
 
 set -x
 
-BASE_GRADLE_ARGS="--profile --continue"
+BASE_GRADLE_ARGS="--profile --no-daemon --continue"
```

Comment:
?

Reply:
I was getting random flakes without this (Exit code 137).  Did a bit of searching online, apparently can be caused if run without `--no-daemon` and Gradle docs specifically recommend running any CI with this option.

https://docs.gradle.org/3.3/userguide/gradle_daemon.html#when_should_i_not_use_the_gradle_daemon